handbook/company/pricing-features-table.yml

```diff
@@ -1133,24 +1094,19 @@
   usualDepartment: IT
   productCategories: [Device management]
   pricingTableCategories: [Device management]
-- industryName: API first by design
-  tier: Free
-  jamfProHasFeature: no
-  jamfProtectHasFeature: no
-  usualDepartment: IT
-  productCategories: [Device management]
-  pricingTableCategories: [Device management]
-- industryName: Targeted device scoping
-  tier: Premium
-  jamfProHasFeature: yes
-  jamfProtectHasFeature: yes
-  usualDepartment: IT
-  productCategories: [Device management]
-  pricingTableCategories: [Device management]
 - industryName: Application deployment
+  description: Deploy applications and security agents on macOS, iOS/iPadOS, Linux, Windows, and Android (coming soon) devices. Additionally, install macOS and iOS/iPadOS apps from the App Store.
   tier: Premium
-  jamfProHasFeature: yes
+  jamfProHasFeature: appleOnly
   jamfProtectHasFeature: no
+  waysToUse:
```

Comment:
Consolidated the descriptions and moreInfoUrls from the above mentioned features.